modules/upgrade/pages/proxy-x.adoc

```diff
@@ -26,180 +19,100 @@ The distribution must be based on {sle}{nbsp}15 SP1, because {productname}{nbsp}
+----
 . Download an ISO image with the installation sources, and mount the ISO image on your server:
 +
 ----
-mkdir /mnt/sle15sp1
-mount -o loop DVD1.iso /mnt/sle15sp1
+mount -o loop DVD1.iso /mnt/
+----
+. Copy everything from the mounted ISO to your local filesystem:
++
+----
+cp -r /mnt/* /srv/images/sle15sp1
 ----
 
 
-.Procedure: Creating an Autoinstallation Distribution
 
+.Procedure: Creating an Autoinstallation Distribution
 . In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Distributions] and click btn:[Create Distribution].
 . In the [guimenu]``Create Autoinstallable Distribution`` section, use these parameters:
 * In the [guimenu]``Distribution Label`` section, type a unique name for the distribution.
-Use only letters, numbers, hyphens (``-``), periods  (``.``), and underscores (``_``), and ensure the name is longer than four characters.
+Use only letters, numbers, hyphens, periods, and underscores, and ensure the name is longer than four characters.
+For example, ``sles_15sp1-x86_64``.
 * In the [guimenu]``Tree Path`` field, type an absolute path to the installation source.
-In this example, this would be [path]``/mnt/sle15sp1``.
+For example, [path]``/srv/images/sle15sp1``.
 * In the [guimenu]``Base Channel`` field, select [systemitem]``SLE-Product-SUSE-Manager-Proxy-4.0-Pool for x86_64``.
 * In the [guimenu]``Installer Generation`` field, select [systemitem]``SUSE Linux Enterprise 15``.
 * In the [guimenu]``Kernel Options`` field, type any options to be passed to the kernel when booting for the installation.
 The [option]``install=`` parameter and the [option]``self_update=0 pt.options=self_update`` parameter are added by default.
 * In the [guimenu]``Post Kernel Options`` section, type any options to  be passed to the kernel when booting the installed system for the first time.
 . Click btn:[Create Autoinstallable Distribution] to save.
 
-image::autoinst_dist_proxy4.png[scaledwidth=80%]
 
 When you have created an autoinstallable distribution, you can edit it by navigating to  menu:Systems[Autoinstallation > Distributions] and selecting the distribution you want to edit.
 
+
+== Create an Autoinstallation Profile
+
 Autoinstallation profiles contain all the installation and configuration data needed to install a system.
 They can also contain scripts to be executed after the installation is complete.
+For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
 
-Use this example profile as a starting point for your own autoinstallation profile:
 
-.Example: Autoinstallation Profile
 
+.Procedure: Creating an Autoinstallation Profile
+. In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Profiles] and upload your autoinstallation profile script.
+For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
+. OPTIONAL: In the ``Kernel Options`` field, type ``autoupgrade=1 Y2DEBUG=1``.
```

Comment:
If you want to create a profile for "upgrade" the option `autoupgrade=1` is required. Only `Y2DEBUG=1` is optional.